### PR TITLE
Goldenrod Museum fix

### PR DESCRIPTION
**Pull request by mauvesea**
Fixes Goldenrod Museum pictures crashing the games. I've used the dba macro instead of fardw in the pic list. I'm not knowledgeable enough to understand why the fardw macro was having a hard issue loading the pic correctly, as that's what's leading to crashes. It only saves 7 bytes, so I'm not sure it's worth it.

### CODE REVIEW

**Comment by SeasickShore** [comment]
I've also changed the pink NPC pal in overcast maps. It was using the fully bright day pal, and that's why it looked off.

File: data/events/paintings/pic_pointers.asm [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-	fardw OmanytePuzzlePic
-	fardw AerodactylPuzzlePic
-	fardw HoOhPuzzlePic
+	dba HoOhPaintingPic
+	dba LugiaPaintingPic
+	dba BellTowerPaintingPic
+	dba KabutoPuzzlePic
+	dba OmanytePuzzlePic
+	dba AerodactylPuzzlePic
+	dba HoOhPuzzlePic
```
Comment (Rangi42) [inline data/events/paintings/pic_pointers.asm:10]:
```suggestion
	fardw HoOhPaintingPic
	fardw LugiaPaintingPic
	fardw BellTowerPaintingPic
	fardw KabutoPuzzlePic
	fardw OmanytePuzzlePic
	fardw AerodactylPuzzlePic
	fardw HoOhPuzzlePic
```

File: engine/gfx/load_pics.asm [hunk 2]
```diff
@@ -276,7 +276,7 @@ GetPaintingPic:
 	ldh [hBGMapMode], a
 	ld hl, PaintingPicPointers
 	ld a, [wTrainerClass]
-	ld bc, 2
+	ld bc, 3
```
Comment (Rangi42) [inline engine/gfx/load_pics.asm:279]:
```suggestion
	ld bc, 2
```

**Comment by Rangi42** [comment]
Thanks for opening this! To fix the Goldenrod Museum paintings I think you just need to remove `inc hl` in `GetPaintingPic` (engine/gfx/load_pics.asm).

**Comment by SeasickShore** [comment]
You're correct, just removing inc hl does the trick as well.

**Pull request closed without merging**